src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -222,8 +222,12 @@ function useSidebarMetadataFetcher(
             if (isEmpty(answer)) {
                 const error = new Error('No suggestions found.');
                 onError(error, ERROR_CODE_EMPTY_METADATA_SUGGESTIONS, { showNotification: true });
+                return [];
             }
 
+            const fields =
+                templateInstances.find(template => template.templateKey === templateKey && template.scope === scope)
```

Comment:
nit: this line is very long and complex. I would recommend extracting out the templateInstances filter into it's own value e.g.
` 
       const template =  templateInstances.find(template => template.templateKey === templateKey && template.scope === scope);
const fields = template?.fields || [] 
`

Reply:
done 👍